src/components/ui/Navigation/Navigation.module.scss

```diff
@@ -72,17 +72,13 @@
   position: relative;
   display: block;
   height: var(--navbar-hamburger-bar-height);
-  transition:
-    transform 0.2s ease-out 0s,
-    background-color 0.2s ease-out 0.2s,
+  transition: transform 0.2s ease-out 0s, background-color 0.2s ease-out 0.2s,
     top 0.2s ease-out 0.3s;
```

Comment:
@roertbb  Did you formatted this part of code without prettier? Because we've got the same config, but it did it differently.

Reply:
Most likely I've missed that but prettier should format that as you pointed out - fixed in https://github.com/akai-org/akai-code/pull/26/commits/4ce10b37eb718df5df1295ce6d0c691c5596bafb.

I guess we'd get to know that once the PR would point to `master`, which would trigger Github Action with linters, however it's much appreciated that you're pointing it out here!